src/log4stash/ElasticClient/IElasticsearchClient.cs

```diff
@@ -13,6 +13,6 @@ public interface IElasticsearchClient : IDisposable
         AuthenticationMethodChooser AuthenticationMethod { get; set; }
         void PutTemplateRaw(string templateName, string rawBody);
         void IndexBulk(IEnumerable<InnerBulkOperation> bulk);
-        IAsyncResult IndexBulkAsync(IEnumerable<InnerBulkOperation> bulk);
+        bool IndexBulkAsync(IEnumerable<InnerBulkOperation> bulk);

```

Comment:
I think no one use this `bool` result and the `IAsyncResult` has been there by mistake (or someone remove the reference without changing to void).
Anyway, I suppose you can change it to `void` 🤷‍♂️